### ISSUE TITLE
[VAMC] Remove link from image in System page Locations listings

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#24010]
## Describe the defect
The Locations listings on system pages have a few accessibility [issues discovered in #17372](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17372). Removing the link from the images is the simplest fix.

[Issues discovered
](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17372)
- Tab order at mobile screensizes does not match visual order. It's fine at desktop screen sizes. In mobile, after tabbing through the facility contact info, focus jumps back up to the linked image which appears before the contact info. 
- Image link has no hover state or anything indicating it is a link until it gets focus. Focus ring does show, but not styled to wrap entire image, so it look like a styling bug.
- Link name may be confusing. Link name comes from the image alt text. 
    - Often the image alt text is the same as the facility name, like `aria-label="Jamaica Plain VA Medical Center"`. Since the facility name is also linked in the listing, this can cause duplicate links, same link name, same link destination. The duplicate links are not immediately adjacent, so not flagged as redundant, but may not be necessary to have both.
    - Alt text does not always provide useful link text or describe where the link goes and it can be verbose. For example, `aria-label="Outside in front of the Primary Care addition at VA Boston's West Roxbury campus.`. In this case, there is different text pointing to the same url and that can be confusing.

## To Reproduce
1. Go to https://www.va.gov/boston-health-care/ at mobile screensize
2. Tab through locations listing
3. Note that the image is visually before the facility title, but that focus moves from facility title to facility contact info and back to the image.
4. Also note the focus ring that appears when tabbing.
5. Tab to move focus off of the image and hover the image.
6. Note that there is no hover state.

## AC / Expected behavior
- [ ] Images in the locations listing are no longer linked 
- [ ] Images in the locations listing are no longer part of the page tab order 

## Screenshots
<details>
<summary>
Locations listing component:
</summary>
<img width="739" height="855" alt="Image" src="https://github.com/user-attachments/assets/221c6ea1-2f19-423e-b26e-fdbf1506e81a" />
</details>

<details>
<summary>
Mobile tab order:
</summary>
<img width="385" height="462" alt="Image" src="https://github.com/user-attachments/assets/38631885-7104-47ab-a233-b980357cf554" />
</details>

<details>
<summary>
Duplicate link names in VoiceOver rotor:
</summary>
<img width="1512" height="982" alt="Image" src="https://github.com/user-attachments/assets/876240a1-0ffd-4583-a5ef-0e4b87e65a6d" />
</details>

<details>
<summary>
VADS media card component in the VA megamenu:
</summary>
<img width="1053" height="571" alt="Image" src="https://github.com/user-attachments/assets/35024e78-18db-47a3-b124-7d95fd3d2a3c" />
</details>

 ## Additional context
- Approach discussed at 4/13 standup. Decided to remove the link from image as the path forward.
  - Cons of removing link from image: user may expect image to be linked since it's a common pattern, linked image provides a larger touch target
  - Pros: simple solution to a few issues that does not require more design time, linked title may be enough, images across VA are rare and usually not linked so not a common VA pattern
- This is not a VADS component, but the [VADS card media variation](https://design.va.gov/components/card/#media) is the closest. 
  - Note that the VADS media card does not have multiple links in the content like the location listings do. And the image is not linked. 
  - You can see this media card in action in the va.gov megamenu and the [Outreach hub](https://www.va.gov/outreach-and-events/outreach-materials/). 
- [Accessible card UI component patterns | UC Berkeley](https://dap.berkeley.edu/websites/accessibility-guidance-developers/card-ui-component#:~:text=A%20large%20clickable%20area%20benefits,as%20required%20for%20linked%20images). Has info on importance of large touch targets and other guidance. 

## Definition of done
- [ ] Development
   - [ ] Code completed 
   - [ ] PR write up complete
   - [ ] Automated tests passed 
   - [ ] Code review requested 
   - [ ] PR approved (meets established quality and security standards)
- [ ] Review instance created/available
  - [ ] Team members notified to start QA process
- [ ] QA Process
  - [ ] A11y review (meets accessibility standards)
  - [ ] Design Review (meets design & UX requirements)
    - [ ] Tested with different devices / screen sizes
  - [ ] Functional/feature review (meets acceptance criteria)
- [ ] QA feedback reviewed/addressed, if any
- [ ] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated

**Comment by qjac** [comment]
Would the fix for the System page (this ticket) also fix the location listing on https://www.va.gov/boston-health-care/locations/? Or does that need to be ticketed separately?
